packages/framework/attributor/src/test/mixinAttributor.spec.ts

```diff
@@ -1,285 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
-
-import { strict as assert } from "node:assert";
-
-import {
-	AttachState,
-	type ICriticalContainerError,
-} from "@fluidframework/container-definitions";
-import { type IContainerContext } from "@fluidframework/container-definitions/internal";
-import { type ConfigTypes, type FluidObject } from "@fluidframework/core-interfaces";
-import { SummaryType } from "@fluidframework/driver-definitions";
-import {
-	type IDocumentStorageService,
-	type ISnapshotTree,
-	type ISequencedDocumentMessage,
-} from "@fluidframework/driver-definitions/internal";
-import {
-	MockLogger,
-	sessionStorageConfigProvider,
-} from "@fluidframework/telemetry-utils/internal";
-import { MockDeltaManager } from "@fluidframework/test-runtime-utils/internal";
-
-import { Attributor } from "../attributor.js";
-import { AttributorSerializer, chain, deltaEncoder } from "../encoders.js";
-import { makeLZ4Encoder } from "../lz4Encoder.js";
-import {
-	type IProvideRuntimeAttributor,
-	createRuntimeAttributor,
-	enableOnNewFileKey,
-	mixinAttributor,
-} from "../mixinAttributor.js";
-
-import { makeMockAudience, makeMockQuorum } from "./utils.js";
-
-type Mutable<T> = {
-	-readonly [P in keyof T]: T[P];
-};
-
-describe("mixinAttributor", () => {
```

Comment:
Don't you want to keep some of these tests?  For the logic that moved to the new DataStore thing

Reply:
Yes, I am looking into it.

Reply:
Add e2e tests.